README.md

```diff
@@ -1 +1,160 @@
-# blogHere
\ No newline at end of file
+## Blog Here
+
+<a name="readme-top"></a>
+<div align="center">
+  <h3><b>Blog Here</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 Blog Here](#blog-here)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [📝 License ](#-license-)
+
+# 📖 Blog Here <a name="about-project">Blog Here</a>
+
+**Blog Here** In this project, I use migration files to build your schema.
+
```

Comment:
**[OPTIONAL]**

- [ ] It would be great if you make your summary more detailed to make your readme descriptive and professional 👍